tests/materials/norsand_test.cc

```diff
@@ -39,13 +39,6 @@ TEST_CASE("NorSand is checked in 3D non-bonded model",
   jmaterial["hardening_modulus"] = 200.0;
   jmaterial["void_ratio_initial"] = 0.85;
   jmaterial["p_image_initial"] = 87014.6;
-  jmaterial["bond_model"] = false;
```

Comment:
Why are they removed from testing?

Reply:
To test that they can be initialized without specifying those `bonded` parameters

Reply:
Are there tests with these values?

Reply:
Test for `false`? The default initialization is `false` anyway right?